### ISSUE TITLE
Does Live transcription support translation?

### ISSUE THREAD

**Issue by hmurovich** [gladiaio/gladia-samples#36]
I have reviewed the documentation and found that it supports translation. However, it seems that this feature may not be available for live transcription scenarios. Could you please confirm whether Gladia  supports translation for live transcription?

**Comment by sboudouk** [comment]
Yes, it does support translation, please check [this link](https://docs.gladia.io/chapters/live-stt/features#translation) for how to use it :)
